rest/models/DashboardTemplate.go

```diff
@@ -184,8 +195,10 @@ type BaseDashboardTemplate struct {
 type BaseTemplates map[AvailableTemplates]BaseDashboardTemplate
 
 type ModuleFederationMetadata struct {
-	Scope      string `json:"scope"`
-	Module     string `json:"module"`
-	ImportName string `json:"importName,omitempty"`
+	Scope      string               `json:"scope"`
+	Module     string               `json:"module"`
+	ImportName string               `json:"importName,omitempty"`
+	Defaults   BaseWidgetDimensions `json:"defaults"`
```

Comment:
As a follow up can we add some documentation to the readme here https://github.com/RedHatInsights/chrome-service-backend/blob/main/docs/dashboard-layouts.md#widgets-mapping to include `Defaults`?